project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.3
+sbt.version=1.4.9
```

Comment:
Does bumping to 1.5.x blow things up?

Reply:
Yeah, and I am not sure exactly why that is. I need to report an issue upstream. 1.4.x version should already help with the benchmarks